### PR TITLE
fix: add Westend with staking to preConfigured

### PR DESCRIPTION
**Pull request by leohar**
<details>
  <summary> Android </summary>

Screenshot
Screenshot

</details>
<details>
  <summary> iOS </summary>

`Westend` is added, but staking is not displayed: https://app.clickup.com/t/86960qyut

</details>

### CODE REVIEW

File: chains/v20/preConfigured/details/e143f23803ac50e8f6f8e62695d1ce9e4e1d68aa36c1cd2cfd15340213f3423e.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
+{
+  "chainId": "e143f23803ac50e8f6f8e62695d1ce9e4e1d68aa36c1cd2cfd15340213f3423e",
+  "name": "Westend",
```
Comment (stepanLav) [inline chains/v20/preConfigured/details/e143f23803ac50e8f6f8e62695d1ce9e4e1d68aa36c1cd2cfd15340213f3423e.json:3]: Shouldn't be a Wetsend (TESTNET)?

**Review by valentunn — APPROVED** [review]

File: scripts/polkadotjs_endpoints_to_preconfigured.py [hunk 2]
```diff
@@ -275,16 +275,18 @@ def process_json_file(file_path):
 
 
 def process_dict_data(file_path, data):
-    if "(SHUTTING DOWN)" not in data.get('name', ''):
+    if "(SHUTTING DOWN)" in data.get('name', '') or "Westend" in data.get('name', ''):
```
Comment (stepanLav) [inline scripts/polkadotjs_endpoints_to_preconfigured.py:278]: That wouldn't work properly, `"Westend" in data` will match with many networks, for example: "Westend BridgeHub (TESTNET)" or "Westend Collectives (TESTNET)"

I would suggest to add "Westend (TESTNET)" as a pattern here and move it to separate array, example:

```python
skip_patterns = ["(SHUTTING DOWN)", "Westend (TESTNET)"]

if any(pattern in data.get('name', '') for pattern in skip_patterns):
    pass
    print(f"Skipped file: {file_path}")
os.remove(file_path)
print(f"Removed file: {file_path}")
```

File: scripts/polkadotjs_endpoints_to_preconfigured.py [hunk 3]
```diff
@@ -275,16 +275,18 @@ def process_json_file(file_path):
...
+        pass
+    else:
         os.remove(file_path)
         print(f"Removed file: {file_path}")
 
 
 def process_list_data(file_path, data):
-    shutting_down_items = [entry for entry in data if "(SHUTTING DOWN)" in entry.get('name', '')]
-    if shutting_down_items:
+    skipped_items = [entry for entry in data if "(SHUTTING DOWN)" in entry.get('name', '') or "Westend" in entry.get('name', '')]
```
Comment (stepanLav) [inline scripts/polkadotjs_endpoints_to_preconfigured.py:286]: there we can use skip_patterns array, if define it on base level in order to decrease repetition

**Comment by valentunn** [comment]
I will merge the PR and lets adjust the script in a separate PR as we need to work with westend

**Pull request merged**